broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -257,4 +268,28 @@ public static Options buildCommandlineOptions(final Options options) {
 
         return options;
     }
+
+    private static void checkMQAlreadyStart(String storePathRootDir) throws IOException {
```

Comment:
What's the relation between `broker` and `store root path`? I would recommend that check whether `store` is available in `MessageStore` module, if `MessageStore` can't start properly, then exit~

Reply:
done